README.md

```diff
@@ -28,4 +28,6 @@ npm install -g truffle ganache-cli
 
 truffle compile
 truffle migrate
-```
\ No newline at end of file
+```
+
+MIT License
```

Comment:
Adding the MIT License to the README is a good practice for open-source projects.